sdk/core/azure-core/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## 1.14.0-beta.2 (Unreleased)
 
 ### Features Added
-
+- Request logs to now include the `accept-range`, `content-range`,`range`,`WWW-Authenticate`, `xx-ms-date`,`x-ms-error-code`,`x-ms-range`  and `x-ms-version` headers.
```

Comment:
consistency nit: add a new line between heading and text (vscode markdown linter also complains about this)

```suggestion

- Request logs to now include the `accept-range`, `content-range`,`range`,`WWW-Authenticate`, `xx-ms-date`,`x-ms-error-code`,`x-ms-range`  and `x-ms-version` headers.
```